app/models/invenio_rdm_record_converter.rb

```diff
@@ -31,6 +31,8 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   def initialize(generic_file=nil)
     return unless generic_file
 
+    $stdout.puts "processing id: #{generic_file.id}, title: #{generic_file.title.first}"
```

Comment:
Will this get logged to the screen, even without the `verbose` and/or `debug` options for the rake task? And with the rake task options, it will be logged to a log file? 

What about limiting logging to only the `generic_file` ones that break?

Reply:
I'm going to check this to make sure. I was having trouble getting output to the screen without the options, but that might have been something wrong with the code itself.

Reply:
It takes a really, really long time for it to start actually processing files. It always throws me off.